### ISSUE TITLE
Support for following stdout

### ISSUE THREAD

**Issue by sbuvaneshkumar** [ansible/awx#7213]
##### ISSUE TYPE
 - Feature Idea

##### SUMMARY
I am able to view the existing job stdout using following command
```
 awx job stdout <job-id>
```
The above command doesn't follow the stdout, but it just exit after displaying the current stdout.
It would be better if we can follow the stdout, adding option like `--follow` would be helpful.

**Comment by ryanpetrello** [comment]
What you want is probably something like `--monitor` (which exists for the launch commands in the CLI).

**Comment by ryanpetrello** [comment]
Duplicate of https://github.com/ansible/awx/issues/6165

You can already do this, instructions at:
https://github.com/ansible/awx/pull/6790

**Comment by sbuvaneshkumar** [comment]
@ryanpetrello I am using the latest version, downloaded via pip3.
```
அ  ~  awx --version   
3.7.0
```
I don't see either `--monitor` or `awx jobs monitor` works:
```
அ  ~  awx -v job stdout 1425307 --monitor
DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): 
<trimmed>
awx: unrecognized arguments: --monitor
```
```
அ  ~  awx jobs monitor 1416669           
usage: awx jobs [-h] action ...

positional arguments:
  action
    list
    get
    delete
    stdout

optional arguments:
  -h, --help  show this help message and exit

awx jobs: argument action: invalid choice: 'monitor' (choose from 'list', 'get', 'delete', 'stdout')
```

**Comment by sbuvaneshkumar** [comment]
I see `--monitor` option can be used only upon launching the job, for existing job, it won't work.

**Comment by ryanpetrello** [comment]
@sbuvaneshkumar,

```
awx --version   
3.7.0
```

This is a Red Hat Ansible Tower version, which is downstream of AWX (and likely doesn't have this feature yet, as it was merge fairly recently):

https://github.com/ansible/awx/pull/6790

If you want to use this, you'll need to wait for Red Hat to release it in their downstream product.

The latest released AWX (11.2.0) *does* have this support:

https://github.com/ansible/awx/releases/tag/11.2.0
https://github.com/ansible/awx/blob/devel/INSTALL.md#installing-the-awx-cli

**Comment by sbuvaneshkumar** [comment]
Thanks for the links :), I was able to install AWX 11.2.0 version and able to use the monitor option.
